src/components/header/index.js

```diff
@@ -61,6 +62,7 @@ export default function Header() {
 								height="28"
 							/>
 						</a>
+						<NavItem />
```

Comment:
Reusing `NavItem` is probably less than ideal, at the very least from a naming perspective, but the prospect of rewriting/renaming everything in this component to be aware of non-link items was... less than appealing.

I hope that's not a problem, but maybe can find the time to rewrite everything if it will be.

Reply:
Agree, that's fine for now 👍